tools/appliance/definitions/systemvmtemplate/configure_systemvm_services.sh

```diff
@@ -107,8 +107,8 @@ EOF
 
   systemctl daemon-reload
   systemctl enable cloud-early-config
-  systemctl disable cloud-passwd-srvr
-  systemctl disable cloud
+  systemctl enable cloud-passwd-srvr
+  systemctl enable cloud
```

Comment:
@wido We may not want to enable the services by default, but only through cloud-early-config's patching?